tests/functional/builtins/codegen/test_extract32.py

```diff
@@ -36,7 +36,7 @@ def extrakt32_storage(index: uint256, inp: Bytes[100]) -> bytes32:
     for S, i in test_cases:
         expected_result = S[i : i + 32] if 0 <= i <= len(S) - 32 else None
         if expected_result is None:
-            assert_tx_failed(lambda: c.extrakt32(S, i))
+            assert_tx_failed(lambda p=(S, i): c.extrakt32(*p))
```

Comment:
note: changes to `assert_tx_failed()` and other fixtures punted to later PR

Reply:
https://github.com/DanielSchiavini/vyper/pull/2